### ISSUE TITLE
A11y: Insufficient contrast for Focus Indicators in Cart Overlay

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#5208]
The focus indicator for the submit buttons to remove a product has insufficient contrast against both the button color AND the background color.

Additionally, the focus indicator for the Shipping button should match the specs for the inputs, per [https://shopware.atlassian.net/browse/NEXT-38789|https://shopware.atlassian.net/browse/NEXT-38789|smart-link]

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-38923**



[created from NEXT-38923, comment 505552]

**Comment by shopwareBot** [comment]
[public]

 > Status Change: **In Review**

 Work on this ticket is complete and is now pending review.



[created from NEXT-38923, comment 511112]

**Comment by shopwareBot** [comment]
[public] 

 > FixVersion Change: **6.6.9.0**

 This issue will be / was solved in Shopware version ***6.6.9.0***.



[created from NEXT-38923, comment 511113]

**Comment by shopwareBot** [comment]
[public]

 > Status Change: **Resolved**

 The ticket has been successfully completed; the solution is implemented and verified.



[created from NEXT-38923, comment 511114]
